packages/ckeditor5-remove-format/src/removeformatcommand.ts

```diff
@@ -8,6 +8,8 @@
  */
 
 import type { DocumentSelection, Item, Schema } from 'ckeditor5/src/engine.js';
+import { isGHSAttributeName, type GHSViewAttributes } from '@ckeditor/ckeditor5-html-support';
```

Comment:
Such cross-package import should not be possible. We used to have a linter rule, but it stopped working. This import will ARAIR break the DLL build.

Reply:
Fixed